job_manager/nmpi-job-manager-app/nmpi/queue/controllers.js

```diff
@@ -158,6 +168,23 @@ angular.module('nmpi')
             });
         };
 
+        $scope.removeTag = function(tag, id){
+            var job = Queue.get({id:id.eId}, function() {
```

Comment:
Is it necessary to get the job again? The job should already be available as `$scope.job`. If my understanding of Angular is correct (it may not be), you could then use `$scope.job.$update()` instead of `Queue.update()`/`Result.update()`